Sources/SwiftParser/Attributes.swift

```diff
@@ -848,6 +848,17 @@ extension Parser {
   }
 }
 
+extension Parser {
+  mutating func parseIsolatedAttributeArguments() -> RawIsolatedAttributeArgumentsSyntax {
+    let (unexpectedBeforeIsolationKind, label) = self.expect(.identifier)
+    return RawIsolatedAttributeArgumentsSyntax(
+      unexpectedBeforeIsolationKind,
+      isolationKind: label,
+      /*unexpectedAfterIsolationKind*/ nil,
+      arena: self.arena)
```

Comment:
I think this is not correctly `swift-format`ed? Could you run `./swift-syntax-dev-utils format`?